public/keymaps/primekb_prime_e_default.json

```
@@ -0,0 +1,12 @@
+{
+  "keyboard": "primeKB/prime_e",
+  "keymap": "default_dc267f9",
+  "commit": "dc267f96c79b82e6b7eb77a36cdfe8b81e601519",
+  "layout": "LAYOUT",
+  "layers": [
+	["KC_ESC", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_DEL", "KC_BSPC", "KC_TAB", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_QUOT", "KC_ENT", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "MO(3)", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_LSFT", "KC_LCTL", "KC_LALT", "MO(2)", "MO(1)", "KC_SPC", "KC_RALT", "KC_RGUI", "KC_RCTL"],
+	["KC_GRV", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_UP", "KC_TRNS", "KC_MUTE", "KC_VOLD", "KC_VOLU", "KC_CAPS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_RIGHT", "KC_SCLN", "KC_BSLS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "BL_TOGG", "BL_STEP", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"],
```

Minor visual improvement: using `KC_RGHT` instead of `KC_RIGHT` will make Configurator display an actual arrow for the key, instead of the text `KC_RIGHT`.

Configurator, for most keycodes, treats the short-form aliases as the expected keycode (though any core QMK keycode will work, and it is possible to add keyboard-specific keycodes in a way that Configurator can compile them).

```suggestion
	["KC_GRV", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_UP", "KC_TRNS", "KC_MUTE", "KC_VOLD", "KC_VOLU", "KC_CAPS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_LEFT", "KC_DOWN", "KC_RGHT", "KC_SCLN", "KC_BSLS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "BL_TOGG", "BL_STEP", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"],
```